utils/utils_test.go

```diff
@@ -63,5 +63,19 @@ func TestAppLoadFromDirectory(t *testing.T) {
 			u.So(t, svcMap["rules"], gc.ShouldNotBeEmpty)
 			u.So(t, svcMap["incoming_webhooks"], gc.ShouldNotBeEmpty)
 		}
+
+		u.So(t, app["environments"], gc.ShouldHaveLength, 5)
+		for _, env := range app["environments"].(map[string]interface{}) {
```

Comment:
its probably best to stick to the `environments, ok := app["environments"].(map[string]interface{})` syntax here and have calls to `u.So(t, ok, gc.ShouldBeTrue)` (we seem a bit inconsistent on that in general in this file)

just if/when that fails, we'll at least get a clean test failure rather than a panic

Reply:
makes sense. I'll do that for all other instances in this file.